### PR TITLE
10183 add module project_priority_report

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: project_priority_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Project Priority Report
+=======================
+
+This module extends the functionality of project report .
```
Comment (elicoidal) [inline project_priority_report/README.rst]: This module extends the functionality of project adding a task report listed by priority.
You can use this report to communicate the priorities to the project members.

File: project_priority_report/README.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (hr_timesheet, project_timesheet', project_issue_sheet)
```
Comment (elicoidal) [inline project_priority_report/README.rst]: No steps required

File: project_priority_report/README.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20project_priority_report%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+
+Contributors
+------------
+
+* Akretion Sébastien BEAU <sebastien.beau@akretion.com>
```
Comment (elicoidal) [inline project_priority_report/README.rst]: This is incorrect.
Should be Sebastien + Eric (AFAIK)

File: project_priority_report/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Project Priority Report',
+    'version': '8.0.1.0.1',
+    'category': 'Human Resources',
+    'depends': [
+        'hr_timesheet',
+        'project_timesheet',
+        'project_issue_sheet',
```
Comment (elicoidal) [inline project_priority_report/__openerp__.py:10]: remove unnecessary dependencies (probably the last one is enough)

**Comment by elicoidal** [comment]
Some details. otherwise LGTM

File: project_priority_report/__openerp__.py (hunk 4 above)
Comment (Rona111) [inline project_priority_report/__openerp__.py:10]: @elicoidal  @siyuancn I have checed ,the project_timesheet was needed in the project_priority_report, and the hr_timesheet is unnecessary.

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Pull request merged**